packages/terra-image/src/Image.jsx

```diff
@@ -49,20 +58,27 @@ const propTypes = {
    * Function to be executed when the image load errors.
    */
   onError: PropTypes.func,
+  /**
+  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
+  */
+  imageFit: PropTypes.oneOf(['cover', 'contain', 'fill', 'scale-down', 'none']),
```

Comment:
because we define these as an enumeration above, want to use object.values?
```suggestion
  imageFit: PropTypes.oneOf(Object.values(ImageFitTypes)),
```